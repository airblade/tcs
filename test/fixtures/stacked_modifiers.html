<button class="dark:md:hover:bg-fuchsia-600 hover:before:bg-pink-500 p-3">
  Save changes
</button>
