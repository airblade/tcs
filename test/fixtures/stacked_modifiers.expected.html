<button class="p-3 hover:before:bg-pink-500 dark:md:hover:bg-fuchsia-600">
  Save changes
</button>
